misk-tailwinds/src/main/kotlin/misk/tailwinds/pages/Navbar.kt

```diff
@@ -0,0 +1,224 @@
+package misk.tailwinds.pages
+
+import kotlinx.html.ButtonType
+import kotlinx.html.TagConsumer
+import kotlinx.html.a
+import kotlinx.html.button
+import kotlinx.html.div
+import kotlinx.html.h1
+import kotlinx.html.img
+import kotlinx.html.li
+import kotlinx.html.main
+import kotlinx.html.nav
+import kotlinx.html.role
+import kotlinx.html.span
+import kotlinx.html.ul
+import kotlinx.html.unsafe
+import misk.tailwinds.Link
+import wisp.deployment.Deployment
+
+data class MenuSection(
+  val title: String,
+  val links: List<Link>
+)
+
+fun TagConsumer<*>.Navbar(
+  appName: String,
+  deployment: Deployment,
+  homeHref: String,
+  menuSections: List<MenuSection> = listOf(),
+  content: TagConsumer<*>.() -> Unit = {}
+) {
+  div("bg-gray-900") {
+    attributes["data-controller"] = "toggle"
+
+    // +"""<!-- Off-canvas menu for mobile, show/hide based on off-canvas menu state. -->"""
+    div("hidden relative z-50 xl:hidden") {
+      role = "dialog"
+      attributes["aria-modal"] = "true"
+      attributes["data-toggle-target"] = "toggleable"
+      attributes["data-css-class"] = "hidden"
+//      +"""<!--
```

Comment:
are these comments or do they do something?

Reply:
plain comments, some left as todos for setting up transitions or CSS changes based on interactivity.